bitnami/mongodb/templates/arbiter/statefulset.yaml

```diff
@@ -155,18 +157,24 @@ spec:
               value: {{ .Values.replicaSetName | quote }}
             - name: MONGODB_ADVERTISED_HOSTNAME
               value: "$(MY_POD_NAME).$(K8S_SERVICE_NAME).$(MY_POD_NAMESPACE).svc.{{ .Values.clusterDomain }}"
-            {{- if .Values.auth.enabled }}
+            {{- if (and .Values.auth.enabled (not .Values.auth.credentialFilePath)) }}
             - name: MONGODB_INITIAL_PRIMARY_ROOT_PASSWORD
               valueFrom:
                 secretKeyRef:
                   name: {{ include "mongodb.secretName" . }}
                   key: mongodb-root-password
+            {{- end }}
+            {{- if .Values.auth.enabled }}
             - name: MONGODB_REPLICA_SET_KEY
               valueFrom:
                 secretKeyRef:
                   name: {{ include "mongodb.secretName" . }}
                   key: mongodb-replica-set-key
             {{- end }}
+            {{- if .Values.auth.credentialFilePath }}
```

Comment:
Can we name as vault credentials? Can we explain it in a read me?

Reply:
Done